packages/gafl-webapp-service/src/pages/order-complete/order-complete.njk

```diff
@@ -3,98 +3,84 @@
-            {% endif %}
-            {% if data.contactMethod === data.howContacted['text'] %}
-                <li class="govuk-body">You should have received a text message with your licence details.</li>
-            {% endif %}
-            {% if data.contactMethod === data.howContacted['none'] %}
-                <li class="govuk-body">Make a note of your licence number and you take it with you when you go fishing.</li>
-            {% endif %}
-            </ul>
-        </div>
+            <p class="govuk-body">
+                It’s important you know your licence number. You can take a screenshot or print out your licence information.
+            </p>
+
+            <form action="licence-information">
+                {{ govukButton({
+                    text: "View your licence information",
+                    classes: "govuk-button--secondary"
+                }) }}
+            </form>
 
-        <div class="govuk-!-margin-top-8 no-print">
-            <h2 class="govuk-heading-m">Before you go fishing</h2>
             {% if data.isPostalFulfilment %}
-                <p class="govuk-body">You can go fishing before you get your licence card. You must be able to give your licence details if asked by an enforcement officer.</p>
-            {% else %}
-                <p class="govuk-body">You can go fishing as soon as your licence starts. You must be able to give your licence details if asked by an enforcement officer.</p>
+                <p class="govuk-body">
+                    If you gave us your email address, you’ll get a message that includes your licence information.
+                </p>
+                <div class="govuk-inset-text">
+                    Make a record of your licence number.
+                </div>
             {% endif %}
-            {{ govukWarningText({
-              text: "Make a record of your licence number",
-              iconFallbackText: "Warning",
-              classes: "govuk-!-margin-top-6"
-            }) }}
-            <p class="govuk-body" class="no-print">Get a <a class="govuk-link" href="{{ data.uri.pdf }}">PDF copy of your licence details</a>. If you gave us your email address, you’ll get a confirmation message that includes your licence number.</p>
-            <p class="govuk-body">You must follow the <a class="govuk-link" href="https://www.gov.uk/freshwater-rod-fishing-rules">rod fishing rules</a> and <a href="https://www.gov.uk/government/collections/local-fishing-byelaws">local fishing byelaws</a>.</p>
-         </div>
 
+            <p class="govuk-body">
+                If an enforcement officer asks to see your licence, you'll need to give them these details.
+            </p>
+
+            <p class="govuk-body">
+                You must follow the <a class="govuk-link" href="https://www.gov.uk/guidance/national-rod-fishing-byelaws-for-england-freshwater-fishing-with-a-rod-and-line" target="_blank">rod fishing byelaws (opens in a new tab)</a>.
```

Comment:
Based on chats that I had with Steve and Fiona, links on this page should open in the same tab, if not, you are missing the `rel="noreferrer noopener”` property